frameworks/Python/weppy/uwsgi.ini

```diff
@@ -0,0 +1,19 @@
+[uwsgi]
+master
+; Increase listen queue used for nginx connecting to uWSGI. This matches
+; net.ipv4.tcp_max_syn_backlog and net.core.somaxconn.
+listen = 65535
```

Comment:
@methane the correct PR is #2228 
I copied that from the flask example: https://github.com/TechEmpower/FrameworkBenchmarks/blob/round-14/frameworks/Python/flask/uwsgi.ini#L5

If this value is not correct then should be changed on every framework, not just this.